app/Http/Controllers/DeviceController.php

```diff
@@ -53,8 +53,7 @@ public function create()
      */
     public function show($id)
     {
-        $device = Device::publicDashData()->findOrFail($id);
-
+        $device = Device::findOrFail($id);
```

Comment:
We must build with all permissions then narrow the scope at the end. To fix the token from being displayed it could be marked as hidden or protected with policy.

Reply:
Thought of maybe having a Developer role that can be used to limit access.

Reply:
Yeah I knew that was why you were displaying the token, but I think we should document somewhere that it needs to be removed/hidden in the release. That way we don't forget about it.

Reply:
Once we have authorization added it will probably be good idea to have a Developer role at least till release.